### ISSUE TITLE
Produce copy&pasteable list of URLs to upladed PyPI packages

### ISSUE THREAD

**Issue by potiuk** [apache/airflow-publish#12]
It would be great if the process of publishing packages produces copy & pasteable list of published packages.

When we upload using twine, twine provides such a list as output (but it's random order), and we use it to copy & paste it to the email sent. There is a comment "sort it before" because otherwise it's very difficult to find provider you are looking for on the list

You can see it in the last email where @eladkal did not sort it :) (and it's especially visible if you upload a big number of providers) https://lists.apache.org/thread/1jwvb6449qx91tf74how5jkm3jyrstjt

So the list of URLS should be nicely sorted and printed, possibly with some markers COPY FROM HERE  , COPY TO HERE

**Comment by potiuk** [comment]
cc: @gopidesupavan

**Comment by eladkal** [comment]
Ah bah. This release was very hard. I missed that.
I wonder if maybe `twine` can just return ordered output?

**Comment by potiuk** [comment]
> I wonder if maybe twine can just return ordered output?

Not that I am aware of . You could likely do `twine .... | sort` as well but that might mess around with the output. And when we switch to Trusted Publishing, we will stop using twine at all, so not worth to fix anything I think. 

@gopidesupavan submitted PRs to change .asf.yaml and make Environments configurable - https://github.com/apache/infrastructure-p6/pull/2020 and that's pretty much the last "blocker" to make it happen.

**Comment by gopidesupavan** [comment]
Yeah good suggestion, currently pypi trusted publisher doesnt output anything urls. so probably we may have to construct urls based on the file names. Good we have `packaging` library parses the wheel file names and gives the version and file as a separate tuple so its easy to construct :) 

Will have a look.

**Comment by potiuk** [comment]
Yeah. It's easy to construct :)

**Comment by gopidesupavan** [comment]
Thinking more of generic way say may be its another summary composable action? or small script. we can produce all the package lists as a job summary that would be easy to look through for release managers (especially not going to inside the job logs). say eg here. it can be verifiable by clicking the package and redirects to PyPI and it can be click copy all the packages under `distributions`  below.

WDYT @potiuk @eladkal ?

<img width="1407" alt="image" src="https://github.com/user-attachments/assets/f5839454-6289-406d-ba01-e374b3ac6a6c" />

<img width="1390" alt="image" src="https://github.com/user-attachments/assets/55025778-1711-4873-a33a-e55663a009c4" />

**Comment by potiuk** [comment]
Yep. Action job summary is a good idea :)

**Comment by potiuk** [comment]
And we can easily format it (GitHub markdown) so that it has both clickable links and easily copy & pasteable list of links like this:

```
https://pypi.org/project/apache-airflow-providers-airbyte/5.0.0rc2
https://pypi.org/project/apache-airflow-providers-alibaba/3.0.0rc2
```

**Comment by potiuk** [comment]
Fixed by #13
